napari/layers/utils/stack_utils.py

```diff
@@ -0,0 +1,188 @@
+import warnings
+from typing import Union, List, Tuple, Dict
+import itertools
+
+import numpy as np
+from napari.layers import Image, Labels
+from napari.utils.colormaps import colormaps
+from napari.layers.image._image_utils import guess_multiscale
+from napari.utils.misc import ensure_iterable, ensure_sequence_of_iterables
+
+
+def split_channels(
+    data: np.ndarray, channel_axis: int, **kwargs,
+) -> List[Tuple]:
```

Comment:
```suggestion
) -> List[FullLayerData]:
```
and then add `from ...types import FullLayerData`